src/py/mat3ra/made/tools/analyze.py

```diff
@@ -231,6 +231,45 @@ def get_atom_indices_with_condition_on_coordinates(
     return selected_indices
 
 
+def get_nearest_neighbors_atom_indices(material: Material, position: List[float] = [0, 0, 0]) -> Optional[List[int]]:
+    """
+    Returns the indices of direct neighboring atoms to a specified position in the material using Voronoi tessellation.
+
+    Args:
+        material (Material): The material object to find neighbors in.
+        position (List[float]): The position to find neighbors for.
+
+    Returns:
+        List[int]: A list of indices of neighboring atoms, or an empty list if no neighbors are found.
+    """
+    structure = to_pymatgen(material)
+
+    voronoi_nn = PymatgenVoronoiNN(tol=0.5)
+    structure.append("X", position, validate_proximity=False)
+    neighbors = voronoi_nn.get_nn_info(structure, len(structure.sites) - 1)
+    neighboring_atoms_pymatgen_ids = [n["site_index"] for n in neighbors]
+    structure.remove_sites([-1])
+
+    all_coordinates = material.basis.coordinates
+    all_coordinates.filter_by_indices(neighboring_atoms_pymatgen_ids)
+    neighboring_atoms_ids = all_coordinates.ids
+
+    return neighboring_atoms_ids
+
+
+def get_center_of_coordinates(coordinates: List[List[float]]) -> List[float]:
```

Comment:
This might be put in `mat3ra-utils` and if you use numpy the logic below could be even simpler

Reply:
Do we use it anywhere else tho?

Reply:
We could use the center of coordinates-related logic in Express, I think. But we can deal with this later. Still, this is probably better put in utils as it doest not act on Material